frontend/src/pages/org/index.ts

```diff
@@ -112,6 +129,48 @@ export class Org extends LiteElement {
           icon: "exclamation-octagon",
         });
       }
+      this.checkIfStorageQuotaReached();
```

Comment:
Since `this.getOrgStorageQuotaReached` is below `await this.getOrg`, but doesn't depend on `this.getOrg` to resolve, you could move this to line 121 above the try-catch block and parallelize requests a bit.